src/resources/locales/en.json

```diff
@@ -158,12 +158,23 @@
         "shelf": "Shelf"
     },
     "publication": {
+        "audio": {
+            "tracks": "Tracks"
+        },
         "cancelledLcp": "This book can't be read because the LCP license has been cancelled.",
         "certificateRevoked": "Certificate is revoked",
         "certificateSignatureInvalid": "Certificate signature is invalid",
         "cover": {
             "img": "cover image"
         },
+        "duration": {
+            "and": "et",
```

Comment:
This should be `and` (currently `et`)

Reply:
https://github.com/readium/readium-desktop/pull/973/commits/e6f8ded19d62fddf5d75efe4b943ae65f613bc7f